StRoot/StEvent/StFttPoint.h

```diff
@@ -57,14 +57,14 @@ inline UChar_t StFttPoint::plane() const { return mPlane; }
 inline UChar_t StFttPoint::quadrant() const { return mQuadrant; }
 inline float StFttPoint::x() const { return mX; } // x position (cm) in local coords.
 inline float StFttPoint::y() const { return mY; } // y position (cm) in local coords.
-inline int StFttPoint::nParentClusters() const { return mClusters.size(); } // Number of points in parent cluster
-inline StFttCluster* StFttPoint::cluster( size_t i ) { return mClusters[i]; } //  Parent cluster of the photon.
+// inline int StFttPoint::nParentClusters() const { return mClusters.size(); } // Number of points in parent cluster
+inline StFttCluster* StFttPoint::cluster( size_t i ) { return mClusters[i % 4]; } //  Parent cluster of the photon.
```

Comment:
Taking i%4 is dangerous.  If a user makes an off-by-one error in the code which calls it (i.e. feeds in a range of valules [1,4] instead of the expected [0,3]), the code will silently work, and return unexpected results.  This type of bug will be difficult to track down.  My recommendation is to either assert(i>=0&&i<=3) OR return nullptr if ( i<0||i>3).  Similar comment for addCluster.

Reply:
Addressed: 3686a6eea3b300c95180bdcee0ad698b5763a88c